azure-pipelines.yml

```
@@ -76,7 +76,7 @@ jobs:
   parameters:
     name: async_await
     displayName: Async / Await
-    rust: nightly-2019-04-22
+    rust: nightly-2019-04-24
```

I believe you'll need: 

```suggestion
    rust: nightly-2019-04-25
```

---

Ah sorry. I didn't see your comment before applying the change myself. It would have saved me time :+1: